resources/hab_build.rb

```diff
@@ -84,8 +84,10 @@ def last_build_env
     live_stream new_resource.live_stream
     sensitive true
   end
+end
 
-  ruby_block "create Habitat project release: #{new_resource.name} #{build_version}" do
+action :publish_to_env do
+  ruby_block "Create Automate project release for Habitat pakage: #{new_resource.name}" do
```

Comment:
Please add `#{build_version}` to the resource name to aid in debugging, as the `ruby_block` doesn't have any other indication as to the version here.